src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala

```diff
@@ -45,15 +52,32 @@ class TasksController(
     asLeader(electionService) {
       authenticated.apply { implicit identity =>
         get {
-          pathEndOrSingleSlash {
-            listTasks()
+          (accepts(MediaTypes.`text/plain`) & pathEndOrSingleSlash) {
```

Comment:
Looks good, but do you think we can do the same thing using marshallers? It should be possible to define one marshaller based on several others using `Marshaller.oneOf`, and akka will do the rest of content negotiation.

Reply:
well I thought about it but the thing is that the data from which one or the other response is created from is not the same - for json endpoint there is much more data fetched so the underlying model it different. That's why I did it this way, not the marshaller way. Does it make sense... ?